contracts/test/v0.8/CronUpkeep_1.test.ts

```diff
@@ -67,7 +66,7 @@ async function createBasicCron() {
   )
 }
 
-describe('CronUpkeep', () => {
+describe('CronUpkeep 1/2', () => {
```

Comment:
not sure how much benefit breaking this down gives, but there's also a cost to it in terms of readability, sharing global variables, beforeEach etc.

If the benefit is marginal then I would say it's better to keep it in one file. Also we are not really controlling the future tests being added so it's likely that some will end up exceeding average time

Reply:
I can revert that commit and we can measure the difference, i think it was a pretty significant time delta

Reply:
It ended up being around 30-40% slower without test splitting https://github.com/smartcontractkit/chainlink/actions/runs/3237575975